calendar_resource/README.rst

```diff
@@ -10,11 +10,26 @@ This module adds new features to facilitate resource management with meetings:
 
 * To attach resources to a meeting, go to `Calendar` => Click on a meeting => `Edit` =>
   `Edit` => add any related resources.
-* To edit or add a resource, go to `Settings` => `Resource` => `Resources`.
+* To edit or add a resource, go to `Settings` => `Resource` => `Resources`. Developer mode
+  has to be enabled for this menu to show.
```

Comment:
That's what I was thinking yeah. Especially since the concept of Resources is becoming more prominent. Should I hide the resources menus in the technical settings then, or just leave them as is?